utils/api/reservation.js

```diff
@@ -41,7 +41,7 @@ export async function updateReservation(props) {
     .from("reservations")
     .update(props.values)
     .eq("id", props.reservationId)
-    .eq("dive_shop_id", props.diveShopId)
+    .eq("dive_shop_id", props.dive_shop_id)
```

Comment:
You are going to create breaking changes here, I suggest you change whatever calls this function